tests/zombienet/start-chopsticks.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const startPlaywright = require('./start-playwright');
+const setupPreconditions = require('./setup-preconditions-api');
+
+// Define the parameters
+const nodeName = 'astar';
+const networkInfo = {
+  nodesByName: {
+    astar: { wsUri: 'ws://localhost:8000/' },
+  },
+};
+
+const args = process.env.BASE_URL || 'https://portal.astar.network/';
+
+const spawnDetached = (cmd) => {
+  const cp = require('child_process').spawn(cmd, [], {
+    detached: true,
+    stdio: 'ignore',
+    shell: true,
+  });
+  cp.unref();
+  return cp;
+};
+
+// Execute the run function with the given parameters
+async function executeRun() {
+  try {
+    const childProcess = spawnDetached(
+      'npx @acala-network/chopsticks@latest xcm -p astar -p shiden'
+    );
```

Comment:
Another way to interact with Chopsticks is to use the `setupContext` helper provided by `chopsticks-testing` module. It requires less setup works. Here is an example: https://github.com/AstarNetwork/Astar/blob/96b1503a4d7a99141b5c08df1091fbcac14746f4/tests/e2e/tests/runtime-upgrade.test.ts#L17

Reply:
Very nice, how would you express HRMP channels between parachain this way?

Reply:
You can setup multiple `setupContext` instances. There are some examples of XCM testing in Acala's e2e test repo.